src/main/java/com/ibm/g11n/pipeline/client/NewTranslationRequestData.java

```diff
@@ -0,0 +1,258 @@
+/*  
+ * Copyright IBM Corp. 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ibm.g11n.pipeline.client;
+
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * <code>NewTranslationRequestData</code> is used for specifying a new
+ * translation request.
+ * 
+ * @author Yoshito Umaoka
+ */
+public class NewTranslationRequestData {
+    private final Map<String, Set<String>> targetLanguagesByBundle;
+
+    private String partner = "IBM";
+    private String name;
+    private String organization;
+    private List<String> emails;
+    private List<String> phones;
+    private EnumSet<IndustryDomain> domains;
+    private List<String> notes;
+    private Map<String, String> metadata;
+    private boolean submit;
```

Comment:
Since this class is returned by `getTranslationRequests()` I actually think you need the full status (as an enum) here, and at least a getter for it.